apps/web/styles/globals.css

```diff
@@ -149,6 +149,9 @@ select:focus {
   --tw-ring-color: var(--brand-color);
   border-color: var(--brand-color);
 }
+/*  https://stackoverflow.com/a/69308190 */
+input[type='text']:focus { box-shadow: none; }
```

Comment:
🙏 nice find :) This has done my head in for a little bit

Reply:
This is causing this weird behaviour https://www.loom.com/share/247906747b714aba99b70dc5b4c2cf08

email field has border on hover but not other text fields. I think border is expected. cc @Udit-takkar